src/components/Permissions/AppPermissions/AppPermissions.jsx

```diff
@@ -147,9 +152,9 @@ const AppPermissions = ({ t }) => {
               />
             )}
 
-            {queryResultRemoteRequests.data.length > 0 && (
+            {filteredRemoteRequests.data?.length > 0 && (
```

Comment:
Since you currently instanciate `filteredRemoteRequests` by doing : 
```
const filteredRemoteRequests =
    queryResultRemoteRequests.data?.length > 0 && ...
```

The variable can be false. So it's weird to have: `false.data?.length`.